osbuild/monitor.py

```diff
@@ -112,6 +118,9 @@ def module(self, module):
         json.dump(options, self.out, indent=2)
         self.out.write("\n")
 
+        # Store the time when the module started.
+        self.timer_start = int(time.time())
+
```

Comment:
```suggestion
        self.timer_start = time.time()

```

I would drop the comment (it is rather obvious, I think ;)). I would also leave the int-conversion to the final consumer, so we avoid hard-coding a granularity.